### PR TITLE
Complete support us section on sponsor page

### PR DESCRIPTION
**Pull request by demiwu96**

### CODE REVIEW

**Review by gh0stl0nely — DISMISSED** [review]

**Comment by gh0stl0nely** [comment]
Nicely done!

**Comment by gh0stl0nely** [comment]
@demiwu96 Can you go to 'src/components/Sponsors/ContactUsSection and do some clean up for me before merging?
1) Delete this block of code in the contactButton class in the ContactUsSection's useStyles hook (empty)
image 
2) Delete the // but leave the line seperating own and library component

image

3) Delete the `display: inline` block css and on the top parent div (the one on top of `display: inline`: block), and add in `marginTop: '70px'` (or whatever value that closely matches Figma's design)? 
image

Thanks tons <3

**Review by gh0stl0nely — DISMISSED** [review]

File: src/components/Sponsors/SupportUsSection.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    divStyle: {
+        backgroundColor: '#4529AE',
+        alignItems: 'center',
+        textAlign: 'center',
+    },
+    title: {
+        color: '#FFFFFF',
+        paddingTop: '96px',
+        marginBottom: '16px',
+        fontFamily: 'Futura',
```
Comment (rishFilet) [inline src/components/Sponsors/SupportUsSection.js]: You'll need to include this in gatsby-config in the fonts array for it to show up:
```
{
            family: `Futura`,
            variants: [`300`, `400`, `500`, `600`, `700`, `800`, `900`],
          }
```

**Review by rishFilet — CHANGES_REQUESTED** [review]

File: src/components/Sponsors/SupportUsSection.js (hunk 1 above)
Comment (gh0stl0nely) [reply src/components/Sponsors/SupportUsSection.js]: @demiwu96 I was talking to Rishi, maybe the font should be kept consistent regardless of screensize and what you saw on Figma may potentially be a mistake from design. Maybe u can use the same font for now.

**Pull request merged**